__tests__/verify-dotnet.ps1

```diff
@@ -0,0 +1,32 @@
+if (!$args[0])
+{
+  throw "Must supply dotnet version argument"
+}
+
+$dotnet = Get-Command dotnet | Select-Object -First 1 | ForEach-Object { $_.Path }
+Write-Host "Found '$dotnet'"
+
+$version = & $dotnet --version | Out-String | ForEach-Object { $_.Trim() }
+Write-Host "Version $version"
+# if ($version -ne $args[0])
```

Comment:
resolving to 3.1 on hosted even though setup 3.0.100 - is it because 3.1 is installed (msi) on the machine and reg keys or gac or something?

Reply:
This probably has to do with multilevel lookup. Since on Windows the default is to look in the configured path and the default install location. ##[set-env name=DOTNET_MULTILEVEL_LOOKUP]0 should set that to not look into the default install location.

Reply:
Also, that functionality and default will change with my PR. So I don't think this is important to fix now.